fuzz/src/slice.rs

```diff
@@ -55,6 +58,34 @@ pub fn slice_canonical_array(array: &ArrayData, start: usize, stop: usize) -> Ar
             .unwrap()
             .into_array()
         }
+        DType::List(..) => {
```

Comment:
If list is canonical we should be able to just call slice on list? Is that not implemented?

Reply:
Ignore me, this is different place